src/main/java/org/opensearch/ad/AnomalyDetectorExtension.java

```diff
@@ -43,7 +44,7 @@ public AnomalyDetectorExtension() {
 
     @Override
     public List<ExtensionRestHandler> getExtensionRestHandlers() {
-        return List.of(new RestCreateDetectorAction(extensionsRunner, this), new RestGetDetectorAction());
+        return List.of(new RestCreateDetectorAction(extensionsRunner, this), new RestGetDetectorAction(), new RestValidateDetectorAction());
```

Comment:
We should add arguments `extensionsRunner, this` for other routes as well

Reply:
@owaiskazi19 that can wait until we do the implementation and know exactly what we need... if we add that then we should also update all the handler code to the new format.